clusterloader2/testing/load/config.yaml

```diff
@@ -1,36 +1,40 @@
+# ASSUMPTIONS:
+# - Underlying cluster should have 100+ nodes.
+# - Number of nodes should be divisible by NODES_PER_NAMESPACE (default 100).
+
 #Constants
 {{$BIG_GROUP_SIZE := 250}}
 {{$MEDIUM_GROUP_SIZE := 30}}
 {{$SMALL_GROUP_SIZE := 5}}
-{{$PODS_PER_NODE := 30}}
 {{$NODES_PER_NAMESPACE := 100}}
-#Variables
+{{$PODS_PER_NAMESPACE := 3000}}
+{{$TEST_THROUGHPUT := 10}}
```

Comment:
Just for reusability of the test - is it possible to somehow overwrite it when calling a test?
i.e. If I would like to run test 5x faster, then the whole config would be exactly the same modulo this line.

Reply:
Depends what you mean by that:
- In load test `TEST_THROUGHPUT` is a constant that determines the execution speed. However, there is no way to set it through the test call.
- There is no easy way to propagate value from call to the test. The one we support is `Nodes`. The question is whether is necessary to support overrides for test variables? (This is tempalting, not shell scripting)
- If time is the only concern here, we can introduce flag to define speed multiplier. This flag would be used by pass to tuning sets. Will it be useful?

Reply:
So what we do now in our tests is:
- we *have* a way to overwrite speed of test (we're using env-var which is hacky, but it's possible)
- I would like to preserve that ability, without touching test config

[e.g. I would like to be able to e.g. run the same test with different speed on different suites]

Reply:
Time is not the only thing. E.g. pod per node is the same thing (we have suite with high-density that again is running the same suite, but with different pods per node).

Reply:
OK. I understand.
I created issue #281 regarding this feature. IMHO we should continue with this PR as it is and I will try to figure out the solution for #281 in the meantime.

Reply:
That's fine